src/humancellatlas/data/metadata/api.py

```diff
@@ -279,6 +279,13 @@ def contact_name(self) -> str:
                       f"Use ProjectContact.name instead.", DeprecationWarning)
         return self.name
 
+@dataclass(init=False)
```

Comment:
```suggestion

@dataclass(init=False)
```

2 blank lines before a class